Adyen/UI/View Controllers/SearchViewController.swift

```diff
@@ -108,18 +108,18 @@ public class SearchViewController: UIViewController {
         childViewController.view.translatesAutoresizingMaskIntoConstraints = false
 
         NSLayoutConstraint.activate([
-            searchBar.leadingAnchor.constraint(equalTo: view.layoutMarginsGuide.leadingAnchor, constant: 0),
-            searchBar.trailingAnchor.constraint(equalTo: view.layoutMarginsGuide.trailingAnchor, constant: 0),
+            searchBar.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 8),
+            searchBar.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -8),
             searchBar.topAnchor.constraint(equalTo: view.layoutMarginsGuide.topAnchor, constant: 0),
 
             noResultsStackView.leadingAnchor.constraint(equalTo: view.layoutMarginsGuide.leadingAnchor, constant: 0),
             noResultsStackView.trailingAnchor.constraint(equalTo: view.layoutMarginsGuide.trailingAnchor, constant: 0),
             noResultsStackView.centerYAnchor.constraint(equalTo: view.layoutMarginsGuide.centerYAnchor),
 
-            childViewController.view.topAnchor.constraint(equalTo: searchBar.layoutMarginsGuide.bottomAnchor, constant: 10),
-            childViewController.view.trailingAnchor.constraint(equalTo: view.layoutMarginsGuide.trailingAnchor, constant: 0),
-            childViewController.view.leadingAnchor.constraint(equalTo: view.layoutMarginsGuide.leadingAnchor, constant: 0),
-            childViewController.view.bottomAnchor.constraint(equalTo: view.layoutMarginsGuide.bottomAnchor, constant: 0)
+            childViewController.view.topAnchor.constraint(equalTo: searchBar.bottomAnchor, constant: 0),
```

Comment:
Not sure if it's applicable here but you think it can be better to use the `safeAreaLayoutGuide`?

Reply:
Try rotating the scree to see if this still looks ok 🤔

Reply:
Looks good in landscape as well - the UITableView does it's "magic" internally 👌